components/style/properties/longhand/box.mako.rs

```diff
@@ -1624,38 +1559,158 @@ ${helpers.predefined_type("scroll-snap-coordinate",
+                    Rotate(theta) => {
+                        let theta = theta.to_computed_value(context);
+                        result.push(computed_value::ComputedOperation::Rotate(0.0, 0.0, 1.0, theta));
+                    }
+                    RotateX(theta) => {
+                        let theta = theta.to_computed_value(context);
+                        result.push(computed_value::ComputedOperation::Rotate(1.0, 0.0, 0.0, theta));
+                    }
+                    RotateY(theta) => {
+                        let theta = theta.to_computed_value(context);
+                        result.push(computed_value::ComputedOperation::Rotate(0.0, 1.0, 0.0, theta));
+                    }
+                    RotateZ(theta) => {
+                        let theta = theta.to_computed_value(context);
+                        result.push(computed_value::ComputedOperation::Rotate(0.0, 0.0, 1.0, theta));
+                    }
+                    Rotate3D(ax, ay, az, theta) => {
                         let ax = ax.to_computed_value(context);
                         let ay = ay.to_computed_value(context);
                         let az = az.to_computed_value(context);
                         let theta = theta.to_computed_value(context);
                         let len = (ax * ax + ay * ay + az * az).sqrt();
                         result.push(computed_value::ComputedOperation::Rotate(ax / len, ay / len, az / len, theta));
                     }
-                    SpecifiedOperation::Skew(theta_x, theta_y) => {
-                        result.push(computed_value::ComputedOperation::Skew(theta_x.to_computed_value(context),
-                                                                            theta_y.to_computed_value(context)));
+                    Skew(theta_x, None) => {
+                        let theta_x = theta_x.to_computed_value(context);
+                        result.push(computed_value::ComputedOperation::Skew(theta_x, computed::Angle::zero()));
+                    }
+                    Skew(theta_x, Some(theta_y)) => {
+                        let theta_x = theta_x.to_computed_value(context);
+                        let theta_y = theta_y.to_computed_value(context);
+                        result.push(computed_value::ComputedOperation::Skew(theta_x, theta_y));
+                    }
+                    SkewX(theta_x) => {
+                        let theta_x = theta_x.to_computed_value(context);
+                        result.push(computed_value::ComputedOperation::Skew(theta_x, computed::Angle::zero()));
+                    }
+                    SkewY(theta_y) => {
+                        let theta_y = theta_y.to_computed_value(context);
+                        result.push(computed_value::ComputedOperation::Skew(computed::Angle::zero(), theta_y));
                     }
                     SpecifiedOperation::Perspective(ref d) => {
```

Comment:
nit: Let's be consistent and remove the `SpecifiedOperation::` prefix from the rest of the branches.